README.md

```diff
@@ -154,15 +233,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+will = Person.new(name: "Will Hawkins", age: 25)
 ```
 
```

Comment:
to "create", use `.create`. If you're gonna use `.new`, you need to also use `.save` to emulate the `.create` method.